templates/docs/examples/layouts/application.html

```diff
@@ -37,7 +37,7 @@
     </div>
   </div>
 
-  <div class="l-main">
+  <main class="l-main">
```

Comment:
Not sure what it has to do with breadcrumbs. Artifact from some other branch?

Reply:
Yes, but it is something we need to do. Is it ok as a drive by?

Reply:
It's fine to keep it, but for future would be good to avoid such things.
This is unrelated change, in a commit called "wip" which will make it quite hard, when looking at the history of changes in the future to understand why it happened.

In this case - it's pretty clear and small change so probably we won't need to track it down in such detail.

Reply:
you're right, reverted